terraform/sessions.tf

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "random_id" "cookie-key-even" {
+  byte_length = 32
+}
+
+resource "google_secret_manager_secret" "cookie-key-even" {
+  provider  = google-beta
+  secret_id = "cookie-key-even"
```

Comment:
It's actually HMAC (odd), Encryption (even)

Reply:
renamed - check ordering on services.sf